app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/ImportFilesScreen.kt

```diff
@@ -197,6 +229,41 @@ private fun ColumnScope.EmailAddressesTextFields(selectedTransferType: () -> Tra
     }
 }
 
+@Composable
+private fun AdvancedOptions(
```

Comment:
"AdvancedOptions" already kind of describes the actual options always present at the bottom of the screen. Maybe this component should have something relating to "BottomSheets" in its name?

Reply:
Except the password one is an alert dialog, not a bottomsheet

Reply:
The options themselves are named `TransferAdvancedSettings()`
So this function should be about `Settings` and not `Options.`

For the full name, maybe something like `AdvancedSettingsDisplay` ?

I think we could also remove the `Transfer` part in `TransferAdvancedSettings()`, it's kinda redundant.

_(Instead, we could rename `Settings` into `Options` if there is too much places where `Options` is already used.)_

Reply:
We should probably go with options to avoid confusion with the general settings

Reply:
I'm gonna handle the renaming in another PR.